tools/gulp-tasks/vf-css.js

```diff
@@ -10,6 +10,7 @@ module.exports = function(gulp, path, componentPath, componentDirectories, build
 
   // Sass and CSS Stuff
   const sass = require('gulp-sass');
+  sass.compiler = require('dart-sass');
```

Comment:
I guess we can't drop the node-sass dependency without changing all the things?

Reply:
Actually - we need that to tell `gulp-Sass` to use `dart-Sass` rather than `node-Sass`

Reply:
So I think node-Sass is a dependency of `gulp-Sass` that'll just 'stick around'. It's not in our package.json?

Reply:
It's a shame as node-sass adds to the project boat and needs compilation. 

Looks like gulp-sass isn't interested in changing that: https://github.com/dlmanning/gulp-sass/issues/731

Stunt know if gulp-dart-sass would be a big change?

Reply:
Looks like gulp-dart-sass is a form of gulp-sass, do may be a drop in change https://github.com/dlmanning/gulp-sass/compare/master...mattdsteele:master

Reply:
I went back to `gulp-Sass` and use of the `compiler` option because it's a more 'mature' npm package. 

I did have `gulp-dart-Sass` working at first